### PR TITLE
Hand over CI pipeline control

### PR DESCRIPTION
**Pull request by lx223**
Changes: 
* modify gitignore to exclude usual C# build results
* modify ci pipeline owning team

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,14 @@ bazel-*
...
+[Dd]ebug/
+[Dd]ebugPublic/
+[Rr]elease/
+[Rr]eleases/
+x86/
+x64/
+bld/
+[Bb]in/
+[Oo]bj/
+[Ll]og/
```
Comment (Helcaraxan) [inline .gitignore]: Could we please add these the same way the current `**/obj/` is handled above and instead simply modify / add
```text
bazel-*
**/[Bb]in/
**/[Ll]og/
**/[Oo]bj/
```
The other directories are typically part of the subtree of these three.

Comment (lx223) [reply .gitignore]: Have updated. PTAL

Comment (lx223) [reply .gitignore]: I may have misunderstood what you meant. I modified ```[Bb]in/
[Oo]bj/
[Ll]og/``` to ```
**/[Bb]in/
**/[Ll]og/
**/[Oo]bj/```

If you were talking about only adding these three, I don't think it'd be enough. For example, Unity could put some plugins under `x86`.

Comment (Helcaraxan) [reply .gitignore]: But this repository does not use Unity. This is C# only and the scripts do not generate anything outside of `obj/` or `bin/` actually. I'm updating the `.gitignore` anyway in #42 so you might just want to drop these changes from the PR and focus only on the owner change.

Comment (lx223) [reply .gitignore]: fair enough. have dropped changes to gitignore

**Pull request merged**